setup.py

```diff
@@ -69,7 +69,9 @@
 install_requires = [
     'Flask>=0.11.1',
     'pyPEG2>=2.15.2',
-    'elasticsearch>=2.0.0,<3.0.0',  # FIXME unpin for Elastic 5.0
+    # NOTE: Overlays have to choose which elasticsearch version they want to
+    # use and pin both elasticsearch and elasticsearch-dsl libraries.
+    'elasticsearch>=2.0.0',
```

Comment:
I would pin both to `'elasticsearch>=2.0.0,<6.0'` and `'elasticsearch-dsl>=2.0.0,<6.0'`. Minimal requirements will work and released version should work together.